bundle/compliance/cis_aws/rules/cis_1_8/data.yaml

```diff
@@ -0,0 +1,67 @@
+metadata:
+  id: 0674190c-677c-5f17-bcc8-f60e913eb9d6
+  name: Ensure IAM password policy requires minimum length of 14 or greater
+  profile_applicability: "* Level 1"
+  description: Password policies are, in part, used to enforce password complexity
+    requirements. IAM password policies can be used to ensure password are at
+    least a given length. It is recommended that the password policy require a
+    minimum password length 14.
+  version: "1.0"
+  rationale: Setting a password complexity policy increases account resiliency
+    against brute force login attempts.
+  audit: >-
+    Perform the following to ensure the password policy is configured as
+    prescribed:
+
+    **From Console:**
+
+    1. Login to AWS Console (with appropriate permissions to View Identity Access Management Account Settings)
+
+    2. Go to IAM Service on the AWS Console
+
+    3. Click on Account Settings on the Left Pane
+
+    4. Ensure "Minimum password length" is set to 14 or greater.
+
+    **From Command Line:**
+
+    ```
+    aws iam get-account-password-policy
+
+    ```
+    Ensure the output of the above command includes "MinimumPasswordLength": 14 (or higher)
+  remediation: >-
+    Perform the following to set the password policy as prescribed:
+
+    **From Console:**
+
+    1. Login to AWS Console (with appropriate permissions to View Identity Access Management Account Settings)
+
+    2. Go to IAM Service on the AWS Console
+
+    3. Click on Account Settings on the Left Pane
+
+    4. Set "Minimum password length" to `14` or greater.
+
+    5. Click "Apply password policy"
+
+    **From Command Line:**
+    ```
+     aws iam update-account-password-policy --minimum-password-length 14
+    ```
+    Note: All commands starting with "aws iam update-account-password-policy" can be combined into a single command.
+  impact: ""
+  default_value: ""
+  references:
+    - https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_passwords_account-policy.html
+    - https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#configure-strong-password-policy
+  tags:
+    - CIS
+    - AWS
+    - CIS 1.8
+    - Identity and Access Management
+  section: Identity and Access Management
+  benchmark:
+    name: CIS Amazon Web Services Foundations
+    version: v1.5.0
+    id: CIS_Amazon_Web_Services_Foundations_Benchmark_v1.5.0
```

Comment:
is this the ID we're going with?

Reply:
good catch!
No, it was auto-generated by the script.
Will change to `cis_aws`

Reply:
I don't see value in that ID once we have those fields in as part of `benchmark` anyway, it just needs to be consistent with how we reference it in the stream from the integration, and how we refer to it in cloudbeat. I'm not sure if we actually came to a decision about changing it, but maybe I missed it